docs/reference/synonyms/apis/put-synonyms-set.asciidoc

```diff
@@ -0,0 +1,150 @@
+[[put-synoyms-set]]
+=== Put Synonyms Set
+
+beta::[]
+
+++++
+<titleabbrev>Put Synonyms Set</titleabbrev>
+++++
+
+Creates or updates a Synonyms Set.
+
+[[put-synonyms-set-request]]
+==== {api-request-title}
+
+`PUT _synonyms/<synonyms_set>`
+
+[[put-synonyms-set-prereqs]]
+==== {api-prereq-title}
+
+Requires the `manage_search_synonyms` cluster privilege.
+
+[[put-synonyms-set-path-params]]
+==== {api-path-parms-title}
+
+`<synonyms_set>`::
+(Required, string)
+Synonyms set identifier to create.
+This identifier will be used by other <<synonym-management-apis>> to manage the synonyms set.
+
+`<body>`::
+(Required, object)
+Contains the definition of the synonym rules that conform the synonyms set:
```

Comment:
What does it mean  "synonym rules that conform the synonyms set:"?

Reply:
Chris had a better suggestion: "Contains the definition of the synonym rules for this synonyms set"